megamek/src/megamek/common/actions/PhysicalAttackAction.java

```diff
@@ -241,8 +242,12 @@ protected static void setCommonModifiers(ToHitData toHit, IGame game,
                 toHit.addModifier(-2, "target is large support tank");
             }
```

Comment:
What about large Fixed-Wing Craft? From what I can tell they don't use LargeSupportTank. I'm also not sure about Support VTOLs being handled by this either (I know this code hasn't changed, but it's the same bit as added code below)